packages/gatsby-theme-carbon/src/components/Header/Header.module.scss

```diff
@@ -121,3 +192,37 @@ header.header {
     border-left: 1px solid $carbon--white-0;
   }
 }
+
+.header :global(a.bx--header__menu-item:hover) {
+  color: $carbon--white-0;
+}
+
+// Style for the subtle border under each item in the headernav sub menu.
+.header
```

Comment:
There's a lot of selectors here that make it tough what the actual intent is. Do we need to use each selector along the way? Why not just `.header .bx--header__menu-item` or better yet, why not a class that we throw onto the menu item ourselves?